web/src/compose_reply.ts

```diff
@@ -267,6 +278,10 @@ export function quote_message(opts: {
 
         compose_ui.replace_syntax(quoting_placeholder, content, $textarea);
         compose_ui.autosize_textarea($textarea);
+
+        if (opts.forward_message) {
+            compose_recipient.open_compose_recipient_dropdown();
+        }
```

Comment:
Hmm, why is this inside `replace_content`? I'd think you would want to do this step without potentially waiting for the server to reply with the original target message markup.

Reply:
The thing is, if compose_recipient.open_compose_recipient_dropdown() is called before the server replies, the dropdown will end up misaligned when the server reply arrives:
![image](https://github.com/user-attachments/assets/e674a9ca-8f7a-40f3-98e9-c92cab93306d)

Is there a way to make it update it's position when the reply arrives? Or will that just make the UI feel buggy, if it's moving around after opening?

Reply:
I'm pretty sure the answer is yes.

@amanagr can you help with this detail?

Reply:
There seems to be a scroll/wheel event listener that can update the position of the dropdown. However I cannot for the life of me figure out where it's beeing called from, and how to call it from the code. Been trying to figure it out for hours now. Some help would be much appreciated!

Reply:
That specific issue could be fixed by just calling `$("#compose_select_recipient_widget")[0]._tippy.popperInstance.update()` when you replace content.